src/axom/quest/tests/CMakeLists.txt

```diff
@@ -156,14 +156,21 @@ if(ENABLE_MPI AND MFEM_FOUND AND MFEM_USE_MPI
             DEPENDS_ON ${quest_tests_depends} mfem
             FOLDER axom/quest/tests
         )
-
-        set(_numMPITasks 1)
-        axom_add_test(
-            NAME ${test_name}
-            COMMAND ${test_name}_test
-            NUM_MPI_TASKS ${_numMPITasks}
-        )
     endforeach()
+
+    set(_numOMPThreads 8) # limit omp parallelism
+    axom_add_test(NAME quest_intersection_shaper
+        COMMAND quest_intersection_shaper_test
+        NUM_MPI_TASKS 1
+        NUM_OMP_THREADS ${_numOMPThreads}
+    )
+    set_tests_properties(quest_intersection_shaper PROPERTIES PROCESSORS ${_numOMPThreads})
```

Comment:
I checked that `axom_add_test` is correctly propagating the `NUM_OMP_THREADS` -- which creates an ENV variable for running the test.

However, BLT does not currently set the `PROCESSORS` test property -- https://cmake.org/cmake/help/latest/prop_test/PROCESSORS.html

I was trying to track down a weird issue where CTest was only using 8 cores/threads for the `quest_intersection_shaper` test (due to NUM_OMP_THREADS), but was not letting anything else run at the same time. 

I found the PROCESSORS property and was trying to see if this would help CTest with scheduling jobs, but it didn't seem to help. (I don't think it hurts, so I didn't remove it)